presto-product-tests/src/main/resources/tempto-configuration.yaml

```diff
@@ -123,6 +123,19 @@ databases:
     jdbc_pooling: true
     table_manager_type: jdbc
 
+  spark:
+      host: spark
```

Comment:
use 2 space ident for consistency with the rest of the file

(we can change in separate commit/pr)